gluten-ut/spark32/src/test/scala/io/glutenproject/utils/clickhouse/ClickHouseTestSettings.scala

```diff
@@ -171,6 +171,9 @@ class ClickHouseTestSettings extends BackendTestSettings {
   enableSuite[GlutenDataFrameRangeSuite]
   enableSuite[GlutenDataFrameSelfJoinSuite]
   enableSuite[GlutenDataFrameSessionWindowingSuite]
+    .exclude("simple session window with record at window start")
```

Comment:
There UTs are disable for the fallback between the two aggregating stages

Reply:
please remember to re-open these uts